pandas/tools/pivot.py

```diff
@@ -382,7 +382,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     columns : array-like, Series, or list of arrays/Series
         Values to group by in the columns
     values : array-like, optional
-        Array of values to aggregate according to the factors
+        Array of values to aggregate according to the factors.
```

Comment:
Only kind of -- in the past, if `values` were specified but `aggfunc` was not, would throw an error. Now that situation defaults to `count`. But if you don't pass `values`, the old behavior persists. It does not change any established non-exception raising behaviors. 

The other option is to not allow users to normalize frequency tables of an arbitrary series of `values`. But seems like this is a pretty normal use case (and satisfies @sinhrks 's request that "I think it is nice if pct_xxx can normalize sum of values if values specified (and all of them are positive)."